src/main/scala/cromwell/Main.scala

```diff
@@ -3,50 +3,104 @@ package cromwell
 import java.io.File
 
 import cromwell.binding._
+import cromwell.engine.SingleWorkflowRunner
 import cromwell.parser.WdlParser.SyntaxError
 import cromwell.server.CromwellServer
-
+import spray.json._
+import scala.util.{Failure, Success}
 
 object Actions extends Enumeration {
-  val parse, validate, run, server = Value
+  val parse, validate, run, inputs, server = Value
 }
 
-
 object Main extends App {
 
   getAction(args.headOption) match {
-    case Some(x) if x == Actions.parse => parseIt(args.tail)
-    case Some(x) if x == Actions.validate => validateIt(args.tail)
-    case Some(x) if x == Actions.run => runIt(args.tail)
-    case Some(x) if x == Actions.server => CromwellServer // Mention it so it gets instantiated
+    case Some(x) if x == Actions.validate => validate(args.tail)
+    case Some(x) if x == Actions.inputs => inputs(args.tail)
+    case Some(x) if x == Actions.run => run(args.tail)
+    case Some(x) if x == Actions.parse => parse(args.tail)
+    case Some(x) if x == Actions.server => CromwellServer
     case None => CromwellServer
     case _ => usageAndExit()
   }
 
-  def parseIt(args: Array[String]): Unit = {
-    if (args.isEmpty) usageAndExit()
-    else println(WdlBinding.getAst(new File(args(0))).toPrettyString)
+  def validate(args: Array[String]): Unit = {
+    if (args.length != 1) usageAndExit()
+    try {
+      val binding = WdlBinding.process(new File(args(0)))
+      println("WDL File is valid")
```

Comment:
I agree about logging in general, though in this particular case this is supposed to be console output for the user, so why clutter it up as a formatted log string?  In fact for a command like this it probably should follow other conventions and just write errors to stderr and then return 0 on success and non-0 otherwise.